### ISSUE TITLE
Enabling Geospatial causes a white screen

### ISSUE THREAD

**Issue by eschoenawa** [SceneView/sceneview-android#140]
When you enable Geospatial the screen turns white. I've attached a screenshot where I've just set `sceneView.geospatialEnabled = true` in the `sample-ar-model-viewer` sample. I'm using v0.9.0 / the current main branch. I'm testing on a Pixel 6 with Android 13, which could be the culprit.
![Screenshot_20220827-221349](https://user-images.githubusercontent.com/7061368/187046578-440dbb5b-89ad-49b8-ab03-754fcd553d1a.png)

**Comment by eschoenawa** [comment]
On my Motorola One Action (Android 11) the screen weirdly goes complete black instead:
![Screenshot_20220827-222210](https://user-images.githubusercontent.com/7061368/187046812-216fd3da-609e-4bb5-a0d1-ca354648b3f9.png)

On another phone (Oneplus 6T, Android 11) the screen glitches out completely with some weird textures:![Screenshot_20220828-000720](https://user-images.githubusercontent.com/7061368/187049648-ce397bf7-9a96-4ee8-821f-5233fa7cbb59.jpg)

**Comment by eguizabal98** [comment]
Maybe you forget the location permission

**Comment by eschoenawa** [comment]
Good idea, but I have granted `ACCESS_FINE_LOCATION` and `ACCESS_COARSE_LOCATION` to the app and still get the same result.

**Comment by yhishi** [comment]
@eschoenawa 
I also had the same problem.
One of the following will solve the problem.

- Declaration in AndroidManifest 
  ```
  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
  ```
- Declaration in AndroidManifest 
  ```
        <meta-data
            android:name="com.google.android.ar.API_KEY"
            android:value="@string/GoogleCloudApiKey" />

        <meta-data
            android:name="com.google.ar.core"
            android:value="required"
            tools:replace="android:value" />
  ```
- requestPermissions `ACCESS_FINE_LOCATION` and `ACCESS_COARSE_LOCATION`

**Comment by kamathis4** [comment]
Not fixed, facing same problem in Redmi note 10 pro
